docker-compose.yml

```diff
@@ -1,7 +1,8 @@
-version: '2.1'
+version: '3.0'
 
 services:
   localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME}"
```

Comment:
Note that this will likely prevent the possibility to scale the service (because container names must be unique). (Not sure if that's a real option in this stack)

Alternatives could be to;

- for someone who wants to use a fixed name, deploy with an extra compose file to override the default
- use a network-scoped alias; https://docs.docker.com/compose/compose-file/

Reply:
The think with local stack I feel is that everything seems to be running on one container.  Also I believe the maintainers may have more insight, but do people tend to run multiple instances of localstack on one machine?  I was assuming it might be too resource heavy?